app/forms/admission_webhook_form.rb

```diff
@@ -0,0 +1,21 @@
+# AdmissionWebhookForm represents a form for AdmissionWebhookController
+class AdmissionWebhookForm
```

Comment:
The idea is to simulate a virtual model and take advantage of some nice things of a model like validation.

Reply:
This will be useful when we introduce the validation of the certificates itself.

Reply:
I am wondering if parts of that this `virtual` model is doing should maybe be part of the `certificate` class, as that handles storage of uploaded certificates?

Then we might have another class to handle the `keys` - as not all views need both.

Reply:
We could `include Certificate`. Would that be enough?

Reply:
As far as I can see the only thing that would be reused, and it would in the controller, is the `Certificate.get_certificate_text` method. Validation for this page has a condition and that differs from the `Certificate` model. 

The reason for that is because the user is not forced to update both certificate and key at once. They can update one or the other or both. So I guess it's fine leaving this way, right? 

We can reuse `get_certificate_text` after we merge #593.

Reply:
Right now you are right - so let's leave it like this.

I assume later on the verification that the certificate is actually a certificate (X509 probably) will be part of certificate.rb, then it might be useful - but I guess we can do that once we work on that feature.